extra/onnx_ops.py

```diff
@@ -66,10 +66,10 @@ def _padding(X, pads=None, auto_pad="NOTSET", axes=None, constant_value=0.):
   constant_padder = Tensor(np.pad(np.zeros(X.shape), np_pads, constant_values=constant_value), dtype=X.dtype)
   return zero_padded + constant_padder
 
-def Pad(x: Tensor, pads: Tensor, constant_value: Tensor=None, axes: Tensor=None, mode="constant"):
+def Pad(x: Tensor, pads: Tensor, constant_value: Tensor=None, axes: Tensor=None, mode="constant", value: float=0.):
   assert mode == "constant"
-  constant_value = 0. if constant_value is None else constant_value.numpy()
-  seq_pads = pads.numpy().astype(np.int32).tolist()
+  constant_value = value if constant_value is None else constant_value.numpy()
+  seq_pads = pads if isinstance(pads, tuple) else pads.numpy().astype(np.int32).tolist()
```

Comment:
Can you make type match?

Reply:
I changed it to convert pads to a list. Is that what you were looking for?

Reply:
No, I mean the function type signature.

Reply:
Alright, fixed.